### PR TITLE
Add readme for fluentd addons

### PR DESCRIPTION
**Pull request by crassirostris**
/cc @steveperry-53

### CODE REVIEW

File: cluster/addons/fluentd-gcp/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Stackdriver Logging Agent
+==============
+
+Stackdriver Logging Agent is a DaemonSet which spawns a pod on each node
+that reads logs, generated by kubelet, container engine and containers
```
Comment (piosz) [inline cluster/addons/fluentd-gcp/README.md]: what is container engine? is kubelet a part of container engine? is docker a part of container engine?

**Review by piosz — CHANGES_REQUESTED** [review]

File: cluster/addons/fluentd-gcp/README.md (hunk 1 above)
Comment (crassirostris) [inline cluster/addons/fluentd-gcp/README.md]: I meant container runtime, thanks for noticing

As for the specifying, what logs fluentd collects, on the one hand I want to give an idea about all sources, on the other hand, I don't want to go into details, explaining that it collects logs from docker daemon, vm creation scripts, salt and so on. It's not a right place for such details IMO.

**Comment by crassirostris** [comment]
@k8s-bot unit test this

**Comment by crassirostris** [comment]
@piosz PTAL

**Comment by crassirostris** [comment]
@k8s-bot cvm gce e2e test this

**Review by piosz — APPROVED** [review]

**Comment by piosz** [comment]
please squash commits

**Comment by crassirostris** [comment]
@piosz I deliberately separated them. They both represent a clearly separable, complete unit of work.

**Comment by piosz** [comment]
Is there any value from having them separately especially when there is gazillion of commits in the repo? Do you want to be able revert one of them independently? Does it help with review?

**Comment by crassirostris** [comment]
@piosz I don't see why not and it looks cleaner to me :) If you feel strong about squashing them, I'm also fine with that

**Pull request merged**